channeldb/messagestore.go

```diff
@@ -0,0 +1,153 @@
+package channeldb
+
+import (
+	"bytes"
+	"encoding/binary"
+
+	"github.com/boltdb/bolt"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/roasbeef/btcd/wire"
+)
+
+var (
+	// basePeerBucketKey is the base key for generating peer bucket keys.
+	// Peer bucket stores top-level bucket that maps: index -> code || msg
+	// Concatenating the message code to the stored data allows the db logic
+	// to properly parse the wire message without trial and error or needing
+	// an additional index.
+	basePeerBucketKey = []byte("peermessagesstore")
+)
+
+// MessagesStore represents the boltdb storage for messages inside
+// retransmission sub-system.
+type MessagesStore struct {
+	// id is a unique identificator of peer.
```

Comment:
What changes in the discovery PR? Peers within the network are identified globally by their public keys. 

In any case, the comment should be replaced with the first sentence of my suggestion: 
> id is a unique slice of bytes identifying a peer.